Needs "--performance-tier Basic"

kemura opened the pull request:
az postgres server create command needs "--performance-tier Basic" option.
I confirmed the cli document was renewed but not this document.

https://docs.microsoft.com/en-us/cli/azure/postgres/server?view=azure-cli-latest#az-postgres-server-create

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @sebastus, has been notified to review your proposed change.

sebastus commented:
@4c74356b41 - thank you very much for your contribution!  
@PRmerger - #signoff

Commit:
Needs "--performance-tier Basic"

az postgres server create command needs "--performance-tier Basic" option.
I confirmed cli document was renewed but not this document.
